draft-packages/avatar/tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+  "extends": "../../tsconfig.dist.json",
+  "compilerOptions": {
+    "composite": true,
+    "allowJs": false,
+    "declaration": true,
+    "declarationDir": "dist",
+    "noEmit": false,
+    "sourceMap": true,
+    "outDir": "dist",
+    "rootDir": "src",
+    "tsBuildInfoFile": "./dist/.cache/tsbuildinfo.json"
```

Comment:
Does this end up in the NPM bundle?

Reply:
It does. I don't know if that's going to be an issue though.... @zioroboco do you have any thoughts? Should I build to `./cache/tsbuildinfo.json"` instead?